extensions/wikia/UserRenameTool/RenameUserLogFormatter.class.php

```diff
@@ -17,88 +17,24 @@ static public function getCommunityUser( $name, $noRedirect = false ) {
 		) ], $name, false );
 	}
 
-	static protected function getCommunityTask( $taskId ) {
-		$title = GlobalTitle::newFromText( 'Tasks/log', NS_SPECIAL, COMMUNITY_CENTRAL_CITY_ID );
-		return Xml::element(
-			'a',
-			[
-				'href' => $title->getFullURL( [ 'id' => $taskId ] ),
-			],
-			"#{$taskId}",
-			false
-		);
-	}
-
-	static public function getCityLink( $cityId ) {
-		global $wgCityId, $wgSitename;
-		$domains = WikiFactory::getDomains( $cityId );
-		if ( $wgCityId == $cityId ) {
-			// Hack based on the fact we should only ask for current wiki's sitename
-			$text = $wgSitename;
-		} else {
-			// The fallback to return anything
-			$text = "[" . WikiFactory::IDtoDB( $cityId ) . ":{$cityId}]";
-		}
-		if ( !empty( $domains ) ) {
-			$text = Xml::tags( 'a', [ "href" => "http://" . $domains[0] ], $text );
-		}
-		return $text;
-	}
-
-	static public function start( $requestor, $oldUsername, $newUsername, $reason, $tasks = [ ] ) {
-		foreach ( $tasks as $k => $v ) {
-			$tasks[$k] = self::getCommunityTask( $v );
-		}
-
-		$text = wfMessage( 'userrenametool-info-started',
+	static private function log( $type, $requestor, $oldUsername, $newUsername ) {
+		$text = wfMessage( $type,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$text". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)

[project-id]: wikia:app